docs/chaos/index.md

```diff
@@ -7,6 +7,37 @@
 
 ![Simmy](../media/simmy-logo.png)
 
+## Usage
+
+<!-- snippet: chaos-usage -->
+```cs
+var builder = new ResiliencePipelineBuilder<HttpResponseMessage>();
+
+// First, configure regular resilience strategies
+builder
+    .AddConcurrencyLimiter(10, 100)
+    .AddRetry(new())
+    .AddCircuitBreaker(new())
+    .AddTimeout(TimeSpan.FromSeconds(5));
+
+// Finally, configure chaos strategies if you want to inject chaos.
+// These should come after the regular resilience strategies.
+
+// 2% of invocations will be injected with chaos
+const double InjectionRate = 0.02;
+
+builder
+    // Inject a chaos latency to executions
+    .AddChaosLatency(InjectionRate, TimeSpan.FromMinutes(1))
+    // Inject a chaos fault to executions
+    .AddChaosFault(InjectionRate, () => new InvalidOperationException("Injected by chaos strategy!"))
+    // Inject a chaos outcome to executions
+    .AddChaosResult(InjectionRate, () => new HttpResponseMessage(System.Net.HttpStatusCode.InternalServerError))
+    // Inject a chaos behavior to executions
+    .AddChaosBehavior(0.001, () => RestartRedisAsync());
```

Comment:
```suggestion
    .AddChaosBehavior(0.001, token => RestartRedisAsync(token));
```

@martincostello , @peter-csala I think we are missing `CancellationToken` here.

Reply:
Please note `Chaos.Behavior.cs` has the same problem.

Reply:
Please also bear in mind that there is no such overload like this currently :D

Reply:
I'll fix it once your PR is merged.